.github/workflows/e2e.yaml

```diff
@@ -62,24 +62,30 @@ jobs:
       matrix:
         k8s: [ v1.27.11, v1.28.7, v1.29.2 ]
     steps:
+    - name: Install kind
+      run: |
+        curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.23.0/kind-linux-amd64
+        chmod +x ./kind
+        sudo mv ./kind /usr/local/bin/kind
     - name: Check out code
       uses: actions/checkout@v4
     - name: Set up Go
       uses: actions/setup-go@v5
       with:
         go-version-file: go.mod
       id: go
-    - uses: actions/download-artifact@v4
+    - name: Download prebuilt konnectivity-server image
+      uses: actions/download-artifact@v4
       with:
         name: konnectivity-server
-    - uses: actions/download-artifact@v4
+    - name: Download prebuilt konnectivity-agent image
+      uses: actions/download-artifact@v4
       with:
         name: konnectivity-agent
-    - name: Install kind
+    - name: Load prebuilt konnectivity images
```

Comment:
can you please tell me what this does ? 

Does it not build a new konnectivity server/agent image per PR which is then used by the e2e tests ?

Reply:
@aojea Could I get you to take a look at this?

Reply:
I was able to fix this by making a new entry in the `Makefile` that doesn't have a dependency on `docker-build`.